lib/search/ui/search_screen.dart

```diff
@@ -0,0 +1,141 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:fluttercon/common/data/models/local/local_session.dart';
+import 'package:fluttercon/common/data/models/local/local_speaker.dart';
+import 'package:fluttercon/common/widgets/resolved_image.dart';
+import 'package:fluttercon/core/di/injectable.dart';
+import 'package:fluttercon/common/utils/router.dart';
+import 'package:fluttercon/search/cubit/search_cubit.dart';
+import 'package:go_router/go_router.dart';
+import 'package:fluttercon/core/theme/theme_colors.dart';
+import '../cubit/search_state.dart';
+import '../models/search_result.dart';
+
+class SearchScreen extends StatelessWidget {
+  const SearchScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (context) => getIt<SearchCubit>(),
+      child: const SearchView(),
+    );
+  }
+}
+
+class SearchView extends StatelessWidget {
+  const SearchView({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Colors.white,
+        elevation: 0,
+        title: Container(
+          padding: const EdgeInsets.symmetric(horizontal: 12.0, vertical: 8.0),
+          decoration: BoxDecoration(
+            color: ThemeColors.lightGrayBackgroundColor,
+            borderRadius: BorderRadius.circular(10),
+          ),
+          child: Row(
+            children: [
+              const Icon(Icons.search, color: ThemeColors.greyTextColor, size: 24),
+              const SizedBox(width: 10),
+              Expanded(
+                child: TextField(
+                  style: const TextStyle(fontSize: 18),
+                  decoration: const InputDecoration(
+                    hintText: 'Search sessions and speakers...',
+                    hintStyle: TextStyle(color: ThemeColors.greyTextColor),
+                    border: InputBorder.none,
+                    contentPadding: EdgeInsets.zero,
+                  ),
+                  onChanged: (query) => context.read<SearchCubit>().search(query),
+                ),
+              ),
+            ],
+          ),
+        ),
+      ),
+      body: BlocBuilder<SearchCubit, SearchState>(
+        builder: (context, state) {
```

Comment:
For uniformity, use the `state.maybeWhen` method. 

Sample: `lib/features/feed/ui/feed_screen.dart:42`

Reply:
We won't need this file [lib/search/ui/search_screen.dart](https://github.com/droidconKE/flutterconKEApp/pull/132/files/db64ebaccaa91908e99241b5ed96613909204903#diff-2e38e0f8cfe387c52232413a31c2630fe746363c14da09bf44b9c6e7352ca171)